packages/webapp/cypress/plugins/index.js

```diff
@@ -16,7 +16,26 @@
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line no-unused-vars
+const axios = require('axios');
+
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  config.env.googleRefreshToken = process.env.GOOGLE_REFRESH_TOKEN;
+  config.env.googleClientId = process.env.REACT_APP_GOOGLE_CLIENTID;
+  config.env.googleClientSecret = process.env.REACT_APP_GOOGLE_CLIENT_SECRET;
+
+  // plugins code ...
+  //const testDataApiEndpoint = `${config.env.apiUrl}`;
+
+
+  on("task", {
+    async "db:tableCleanup"() {
+      // clean up the database tables
+      const { data } = await axios.post('http://localhost:5001/testData/tableCleanup');
```

Comment:
Hardcoding the port will probably be a problem. Also I'm concerned about this working in Docker containers.

Reply:
I will fix the hardcoding of the port, not sure what i need to do re: making it work in docker containers